cli/compose/loader/loader.go

```diff
@@ -444,6 +444,12 @@ func LoadVolumes(source map[string]interface{}) (map[string]types.VolumeConfig,
 			if volume.External.Name == "" {
 				volume.External.Name = name
 				volumes[name] = volume
+			} else {
+				logrus.Warnf("volume %s: volume.external.name is deprecated, please use volume.name", name)
```

Comment:
Maybe "is deprecated in favor of", to avoid using "please" and also avoid the comma splice.